data/helpers.d/mysql

```diff
@@ -90,3 +90,46 @@ ynh_mysql_create_user() {
 ynh_mysql_drop_user() {
     ynh_mysql_execute_as_root "DROP USER '${1}'@'localhost';"
 }
+
+# Create a database, an user and its password. Then store the password in the app's config
+#
+# Password of database will be stored in db_pwd's variable.
+# And store with the name mysqlpwd into the app config.
+#
+# usage: ynh_mysql_setup_db user name
+# | arg: user - Owner of the database
+# | arg: name - Name of the database
+ynh_mysql_setup_db () {
+	db_pwd=$(ynh_string_random)	# Generate a random password
+	ynh_mysql_create_db "$2" "$1" "$db_pwd"	# Create the database
+	ynh_app_setting_set $app mysqlpwd $db_pwd	# Store the password in the app's config
```

Comment:
What do you think about the name of password's variable.
Maybe it's better to use the same name for the variable and for the value name in the settings file.
We can use 'mysqlpwd' only